web-dev.md

```diff
@@ -63,15 +64,25 @@
 
 - Hitesh Chaudhary [JavaScript Playlist](https://www.youtube.com/playlist?list=PLC3y8-rFHvwgg3vaYJgHGnModB54rxOk3)
 
+- DevEd [JavaScript & SVG](https://www.youtube.com/channel/UClb90NQQcskPUGDIXsQEz5Q)
+
 ### Documentation
 
 - [JavaScript](https://developer.mozilla.org/en-US/docs/Web/javascript)
 - [React](https://reactjs.org/docs/getting-started.html)
 - [Angular](https://angular.io/docs)
 - [Node](https://nodejs.org/en/docs/)
 - [Deno](https://doc.deno.land/https/github.com/denoland/deno/releases/latest/download/lib.deno.d.ts)
+- [Chart.js](https://www.chartjs.org/docs/latest/)
```

Comment:
Can we further divide Documentation into Framework and Libraries?

Reply:
Sure i will do it